Library/ViewModels/LiveStreamDiscoveryViewModel.swift

```diff
@@ -0,0 +1,99 @@
+import KsApi
+import LiveStream
+import Prelude
+import ReactiveSwift
+import Result
+
+public protocol LiveStreamDiscoveryViewModelInputs {
+  /// Call from parent controller when this view is shown to the user.
+  func isActive(_ active: Bool)
```

Comment:
might this be more in line with our format if it were `configureWith(isActive: Bool)` since it's called the moment before it is presented?